setup.py

```diff
@@ -75,7 +75,9 @@
         'Programming Language :: Python :: 3.5',
     ],
     install_requires=[
+        'bcrypt>=3.0.0',
         'cryptography>=1.1',
+        'pynacl',
```

Comment:
Would love at least some pinning here; latest `pynacl` seems to be `1.1.2`. What's good - `pynacl>=1.0`, `>=1.1`, other?